blog/2025-03-21-components-testing.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Automated Component Testing
+slug: automated-component-testing
+authors: [goruha]
+tags: [testing, components, automation]
+date: 2025-03-21
+---
+
```

Comment:
This post is out of date. See: https://www.notion.so/cloudposse/Announce-Component-Testing-1bd56d4f15598088a0ddda4e197b5aff?pvs=4